assembl/static2/js/app/components/administration/resourcesCenter/editResourceForm.jsx

```diff
@@ -0,0 +1,159 @@
+// @flow
+import classnames from 'classnames';
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n, Translate } from 'react-redux-i18n';
+import { Button, FormGroup, OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+import {
+  deleteResource,
+  updateResourceDocument,
+  updateResourceEmbedCode,
+  updateResourceImage,
+  updateResourceText,
+  updateResourceTitle
+} from '../../../actions/adminActions/resourcesCenter';
+import FileUploader from '../../common/fileUploader';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+import { getEntryValueForLocale } from '../../../utils/i18n';
+
+type EditResourceFormProps = {
+  documentFilename: string,
+  documentUrl: string | File,
+  embedCode: string,
+  handleDocumentChange: Function,
+  handleEmbedCodeChange: Function,
+  handleImageChange: Function,
+  handleTextChange: Function,
+  handleTitleChange: Function,
+  id: string,
+  imgMimeType: string,
+  imgUrl: string | File,
+  locale: string,
+  markAsToDelete: Function,
+  order: number,
+  text: string,
+  title: string
+};
+
+const deleteResourceTooltip = (
```

Comment:
Please can you add this component in `components/common/tooltips.jsx`.
We should refactor a little bit `tooltips.jsx` to make it more generic.

Reply:
In fact, I did it first and then I talked about it with Vincent and moved it to this file. Why do you want tooltips in `tooltips.jsx` ? Most of the tooltips will be specific to one use case, aren't they?

Reply:
done, it will be easier to refactor